Device.js

```diff
@@ -1,7 +1,5 @@
-var { NativeModules } = require('react-native');
-
+var { NativeModules, Dimensions } = require('react-native');
```

Comment:
Yeah no problem. In most cases hound gives good feedback, here this seems to be not the case.